Alfentanil-Model.json

```diff
@@ -1,8102 +1,8297 @@
-{
```

Comment:
Lot of changes here due to the snapshot conversion.
To see the **significant** changes (consolidation of protein expression profiles) - refer to this commit:
https://github.com/Open-Systems-Pharmacology/Alfentanil-Model/pull/24/commits/00beb5b35a8f0a5379d183769e76be10e79ff75e